samples/e2e/EventHubsCaptureEventGridDemo/WindTurbineDataGenerator/packages.config

```diff
@@ -3,7 +3,6 @@
   <package id="Azure.Core" version="1.5.0" targetFramework="net461" />
   <package id="Azure.Messaging.EventHubs" version="5.2.0" targetFramework="net461" />
```

Comment:
We should upgrade this to 5.4.0 while we're here;  no code changes should be needed.

Reply:
Thanks @jsquire, updated.